docs/docs/unit-testing.md

```diff
@@ -217,50 +217,24 @@ by running `npm test -- -u`.
 
 ## Using TypeScript
 
-If you are using TypeScript, you need to make a couple of small changes to your
-config. First install `ts-jest`:
+If you are using TypeScript, you need to make two small changes to your
+config.
 
-```shell
-npm install --save-dev ts-jest
+Update the transform in `jest.config.js`, like so:
+
+```js:title=jest.config.js
+    "^.+\\.[jt]sx?$": "<rootDir>/jest-preprocess.js",
 ```
 
```

Comment:
I'd also add an example here to remove any ambiguity:
````suggestion

For example, if your test utilities are centrally located in a `lib/` directory in your Gatsby site:

```js:title=jest.config.js
    "^.+\\.[jt]sx?$": "lib/jest-preprocess.js",
```
````

Reply:
The test files don't have to live inside the lib folder. In my project where I've applied the changes all my components are in a separate folder, and all tests live inside this folder...

Reply:
I think there's a misunderstanding here. The user isn;t supposed to replace `<rootDir>` with their own path: the jest config expects the literal string `<rootDir>`. which is replaced *by Jest* with the root directory of the project. e.g. see https://github.com/gatsbyjs/gatsby/blob/master/jest.config.js#L37

Reply:
Ok, that is definitely confusing as angle brackets often denote "replace this" in docs. You can disregard my example, but we should clarify that the literal string `<rootDir>` needs to be included.